### PR TITLE
test(robot-server): RSS-100 persistence snapshot 6.1

### PR DESCRIPTION
**Pull request by y3rsh**
# Overview

6.1 persistence snapshot.

## Risk assessment

None, test only.

### CODE REVIEW

File: robot-server/tests/integration/persistence_snapshots/README.md [hunk 1]
```diff
@@ -21,13 +21,92 @@ The JSON protocols were chosen to cover a wide breadth of Protocol Engine comman
...
+
 ### v6.2.0
 
-This snapshot comes from v6.2.0 on a real non-refresh robot. The robot was power cycled following the successful execution of both protocols.
+This snapshot comes from v6.2.0 on a real non-refresh robot. The robot was restarted following the successful execution of both protocols.
+
+The 2 protocols are to provide basic coverage of a python and json protocol. Each protocol has 1 successful analysis and run.
 
-The 2 protocols are to provide basic coverage of a python and json protocols. Each protocol has 1 successful analysis and run.
+**NOTE** this db will cause a downgrade migration issue if loaded on a robot with 6.1. The error in the logs does not make it clear but an issue would be expected as there is a module in the DB not released until 6.2 (Thermocycler gen 2)
```
Comment (SyntaxColoring) [inline robot-server/tests/integration/persistence_snapshots/README.md]: In the general case, I would expect potential downgrade problems with any combination of versions where the persistence directory is newer than the server. So I don't know if we need to bring this up here.

I agree that the errors are confusing and that this is a problem. I've just ticketed this as RSS-162. Let's move this log over there?

Comment (SyntaxColoring) [reply robot-server/tests/integration/persistence_snapshots/README.md]: Aside: What makes you think that this particular one is associated with the Thermocycler GEN2? From my quick skim of the logs, it looks like something with `PipetteNameType`?

**Review by SyntaxColoring — APPROVED** [review]
Awesome, thanks! One request for moving those error logs to a ticket instead of the readme. Other than that, this looks good to merge.

File: robot-server/tests/integration/persistence_snapshots/README.md (hunk 1 above)
Comment (y3rsh) [reply robot-server/tests/integration/persistence_snapshots/README.md]: I did not troubleshoot it or think much on what I wrote.  I will move this out but what I want to say is
> I am not surprised that an error occurred on downgrade of the robot from a 6.2 database that contained a module not released until 6.2 but this error may not be about that.

My understanding is that we are not supporting downgrade migration anyway, but I wanted to denote that we could get that state with this snapshot 😄

Comment (y3rsh) [reply robot-server/tests/integration/persistence_snapshots/README.md]: https://opentrons.atlassian.net/browse/RSS-163 created to house the error for posterity.

**Pull request merged**